plugins/modules/aws_kms.py

```diff
@@ -821,18 +821,19 @@ def create_key(connection, module):
     if module.params.get('policy'):
         params['Policy'] = module.params['policy']
 
+    if module.check_mode:
```

Comment:
This should go at the very top of the function.

Reply:
I kind of like to run as much of the module as possible, up until the thing that actually does the change. Is this unconventional with check mode?

Reply:
depends on the author, I tend to keep it as 'late' as possible too.

Reply:
In this particular case those few lines don't actually do anything, so I prefer having it at the beginning to make it clear that the function doesn't do anything.

Reply:
Made this change.